components/script/dom/gpucanvascontext.rs

```diff
@@ -105,6 +105,9 @@ pub struct GPUCanvasContext {
     channel: WebGPU,
     /// https://gpuweb.github.io/gpuweb/#dom-gpucanvascontext-canvas
     canvas: HTMLCanvasElementOrOffscreenCanvas,
+    #[ignore_malloc_size_of = "manual writing is hard"]
+    /// https://gpuweb.github.io/gpuweb/#dom-gpucanvascontext-configuration-slot
```

Comment:
To prevent a rustdoc warning: 

```suggestion
    /// <https://gpuweb.github.io/gpuweb/#dom-gpucanvascontext-configuration-slot>
```